library/src/cache/patch_manager.rs

```diff
@@ -810,6 +845,95 @@ mod next_boot_patch_tests {
 
         Ok(())
     }
+
+    // The constant values below were generated by taking the hash of the patch file
+    // using openssl to sign it with a private key.
```

Comment:
What is "the patch file"? here?  "MESSAGE"?

Reply:
Improved these a bit